slave/build_images.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python
+
+import sys
+import time
+from uuid import UUID
+from datetime import datetime
+import yaml
+from common import (
+    wait_for_image, rackspace_is_ready, load_manifest
+)
+
+from libcloud.compute.base import NodeImage
+from libcloud.compute.deployment import ScriptDeployment
+
+# Watch out for dodgy providers that mangle your tag names (I'm looking *right*
+# at you Rackspace).
+BASE_NAME_TAG = 'base_name'
```

Comment:
If we are going to change this, https://github.com/ClusterHQ/build.clusterhq.com/blob/master/flocker_bb/ec2.py#L219 will need to be adjusted as well.